app/src/main/res/values/strings.xml

```diff
@@ -116,10 +116,10 @@
     <string name="list_size_summary">Text size in the element list</string>
     <string name="loading_database">Loading database…</string>
     <string name="lowercase">Lower-case</string>
-    <string name="maskpass_title">Mask password</string>
-    <string name="maskpass_summary">Hide passwords by default.</string>
+    <string name="maskpass_title">Hide passwords</string>
+    <string name="maskpass_summary">Mask passwords (***) by default.</string>
     <string name="menu_about">About</string>
-    <string name="menu_change_key">Change Master Key</string>
+    <string name="menu_change_key">Change Master Keyfile</string>
```

Comment:
Open a dialog to change all the master key (password, keyfile and others in the future), not only the keyfile.